client/src/modules/shopping-list/model/actions.js

```diff
@@ -1,45 +1,89 @@
 import { ENDPOINT_URL } from 'common/constants/variables';
 import { getData, postData } from 'common/utils/fetchMethods';
 import { ShoppingListActionTypes } from './actionTypes';
-
+import { MessageType as NotificationType } from 'common/constants/enums';
+import { createNotificationWithTimeout } from 'modules/notification/model/actions';
 // Action creators
-export const fetchProductsError = err => ({
+const fetchProductsFailure = errMessage => ({
   type: ShoppingListActionTypes.FETCH_PRODUCTS_FAILURE,
-  err
+  errMessage
 });
-export const recieveProducts = json => ({
-  type: ShoppingListActionTypes.FETCH_PRODUCTS_REQUEST,
+const fetchProductSuccess = json => ({
+  type: ShoppingListActionTypes.FETCH_PRODUCTS_SUCCESS,
   products: json
```

Comment:
leave products here just for now, and in other new created actions - make payload